store/modules/playback/actions.js

```diff
@@ -217,13 +220,11 @@ export default {
 
     try {
       if (no_barcode == true) {
-        result = await Vue.axios.get(
-          `${baseurl}/${endpoint}?time_index=${time_index}`
-        );
+        const whole_url_no_barcode = `${baseurl}/${endpoint}?time_index=${time_index}`;
+        result = call_axios_get_from_vuex(whole_url_no_barcode, context);
```

Comment:
can you separate out the logic of generating the `whole_url` from the invocation of `call_axios_get_from_vuex` so that we only need to have a single line of code calling `call_axios_get_from_vuex`?

Reply:
Completed